python-polars/scanning_data.py

```diff
@@ -0,0 +1,31 @@
+import pathlib
+from downloads import download_file
+import polars as pl
```

Comment:
Another small quibble. Let's do the local import after the third-party import:

```suggestion
import pathlib
import polars as pl
from downloads import download_file
```